app.json

```diff
@@ -0,0 +1,48 @@
+{
```

Comment:
This file is the Heroku descriptor, right? I think we can ditch it in the micros branch.